vite.config.ts

```diff
@@ -1,7 +1,37 @@
-import { defineConfig } from "vite";
+import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react";
 
+function getFriendbotUrl(mode: string) {
+  // See https://developers.stellar.org/docs/tools/quickstart/faucet for more information
+  // on friendbot and the local faucet
+  const env = loadEnv(mode, process.cwd(), '')
+  switch (env.PUBLIC_STELLAR_NETWORK) {
+    case "local":
+      return "http://localhost:8000";
```

Comment:
Shouldn't this be `/friendbot`?

```suggestion
      return "http://localhost:8000/friendbot";
```

Reply:
^^ what zach said! we're calling the /friendbot endpoint, and rather than rewrite things, we pass that on through and so the friendbot URL is really about the host.

Reply:
But in other environments we don't want the `/friendbot` path. Will this logic make us try to hit `https://friendbot.stellar.org/friendbot` when `PUBLIC_STELLAR_NETWORK === 'testnet'`?

Would it be easier to set `PUBLIC_STELLAR_FRIENDBOT_URL` as the env variable and remove the proxy?